lib/tasks/import_images.rake

```diff
@@ -0,0 +1,11 @@
+namespace :import_images do
+  desc "Imports placeholder images using the Faker gem"
+  task generate_profile_pictures: :environment do
+  	# Randomly grab half of all Authors, and set their profile images to Faker data;
```

Comment:
@BrianAllenGit noticing tab characters here which mis-aligns your code with rails generated code.

Use two-spaces indenting to match the Rails & Ember frameworks.